datasetIngestor/checkMetadata.go

```diff
@@ -251,7 +251,7 @@ func GatherMissingMetadata(user map[string]string, metaDataMap map[string]interf
 	}
 
 	// for raw data add PI if missing
-	if err := addPrincipalInvestigatorFromProposal(user, metaDataMap, client, APIServer, accessGroups); err != nil {
```

Comment:
why is this removed?

Reply:
![image](https://github.com/user-attachments/assets/db581940-4039-4547-bd4b-bf2a32214833)
it wasn't not removed, GetProposal doesn't need accessGroups so it was removed from the parameter list